Assets/Scripts/SpikeScript.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/***************************************************************************************************
+ * Requires that the player object to have the tag "Player". Each spike will receive a BoxCollider 
+ * and is set on awake to a specific offset and size. The player OnCollision with a spike will 
+ * always be pushed at a 45 degree angle. Intensity is multiplied by the signed Vector. 
+ * Do note, if intensity > 5, then knockback will be greater than the players actual jump height.
+ * Also, this script doesnt disable the players jumping ability. So the player can jump and move
+ * while being knockedback.
+ **************************************************************************************************/
+public class SpikeScript : MonoBehaviour
+{
+    [SerializeField] private float intensity;
+    [SerializeField] private float spikeDamage;
+    private PolygonCollider2D poly;
+
+    void Awake() {
+        poly = gameObject.AddComponent<PolygonCollider2D>();
```

Comment:
Instead of adding the collider through the script, it would make it easier to control if you just add it to the prefab.

Reply:
https://github.com/AztecGameLab/LongTermGame5/pull/164/commits/f32b1371c5e1c9df76267ba78c553f4883e3b151